tools/ci-cdk/README.md

```diff
@@ -5,6 +5,30 @@ continuous integration.
 
 The `cdk.json` file tells the CDK Toolkit how to synthesize the infrastructure.
 
+## Canary local development
+
+Sometimes it's useful to only deploy the the canary resources to a test AWS account to iterate
+on the `canary-runner` and `canary-lambda`. To do this, run the following:
+
+```bash
+npm install
+npm run build
+npx cdk --app "node build/bin/canary-only.js" synth
+npx cdk --app "node build/bin/canary-only.js" deploy --outputs-file cdk-outputs.json
+```
+
+From there, you can just point the `canary-runner` to the `cdk-outputs.json` to run it:
+
+```bash
+cd canary-runner
+cargo run -- --sdk-version 0.9.0 --musl --cdk-outputs ../cdk-outputs.json
```

Comment:
Can we add a note somewhere about updating this version number on release or replacing it with something like `<version>`?